### PR TITLE
Move meta tags for document rendering

### PR DESCRIPTION
**Pull request by rlgreen91**
Move meta tags used for document rendering in the head element to ensure
consistent document rendering.

In your PR did you:

  - [X] Include a description of the changes?
  - [ ] Mention the issue the PR addresses?
  - [ ] Include screenshots of any changes to the UI?
  - [ ] Add and/or update specs for your code?
  - [ ] Update the release tasks script to reflect tasks that should run when deploying to staging or production?

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    # These elements must come first to consistently ensure proper document rendering.
```
Comment (mnyon-grandkru) [inline app/views/layouts/application.html.erb:4]: Is this a ruby comment in an HTML section?

Comment (rlgreen91) [reply app/views/layouts/application.html.erb:4]: It is...but it's in the head element so technically fine?  I'll update it anyways.

Comment (mnyon-grandkru) [reply app/views/layouts/application.html.erb]: I wasn't sure of that. That's okay then

**Review by mnyon-grandkru — APPROVED** [review]
LGTM

**Pull request merged**